packages/Pagination/src/Pagination.js

```diff
@@ -0,0 +1,107 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { ShirtSizes } from "@paprika/helpers/lib/customPropTypes";
+import ArrowItem from "./components/ArrowItem/ArrowItem";
+import CurrentPageItem from "./components/CurrentPageItem/CurrentPageItem";
+import ElipsisItem from "./components/EllipsisItem/ElipsisItem";
+import PageItem from "./components/PageItem/PageItem";
+import Divider from "./components/Divider";
+import * as sc from "./Pagination.styles";
+
+const propTypes = {
+  /** The number of current active page */
+  currentPage: PropTypes.number.isRequired,
+  /** Length of the pagination (the number of other pages will be displayed around active page). */
+  length: ShirtSizes,
+  /** Callback to be executed when current page is changed. */
+  onClickPage: PropTypes.func.isRequired,
```

Comment:
we don't have another click event I would say its safe to just use `onClick` without the subfix.